src/rmplan/rmplan.ts

```diff
@@ -411,4 +454,30 @@ program
     }
   });
 
+  // --- Configuration Loading ---
```

Comment:
This looks weird, just do it in the command or see if commander has a way to parse top-level options.